### PR TITLE
For old bugs with low priority and low severity assigned to inactive people, unassign without needinfoing the triage owner

### PR DESCRIPTION
**Pull request by marco-c**
I'm not sure if we actually want to do this, maybe we can do it in case we get complaints about the tool.

### CODE REVIEW

**Comment by marco-c** [comment]
Speaking with @suhaibmujahid on Matrix, we came up with a slightly different idea: not needinfo if the bug hasn't been touched for a while and it has a low priority and low severity. Indeed, if it has a high priority or high severity, we want to needinfo so the triage owner can set it to lower or close the bug if it is no longer relevant.

File: auto_nag/scripts/assignee_no_login.py [hunk 1]
```diff
@@ -64,14 +67,35 @@ def get_mail_to_auto_ni(self, bug):
...
             return None
 
+        # Avoid to ni if the bug was last touched many years ago and has low priority and low severity.
+        # It's not paramount for triage owners to make an explicit decision here, it's enough for them
+        # to receive the notification about the unassignment from Bugzilla via email.
+        last_change = dateutil.parser.parse(bug["last_change_time"]).replace(
+            tzinfo=None
+        )
+        if (
+            last_change < datetime.datetime.utcnow() - relativedelta(years=3)
```
Comment (suhaibmujahid) [inline auto_nag/scripts/assignee_no_login.py:77]: We could move up `datetime.datetime.utcnow() - relativedelta(years=3)` to the constructor.

**Review by suhaibmujahid — APPROVED** [review]

File: auto_nag/scripts/assignee_no_login.py (hunk 1 above)
Comment (marco-c) [reply auto_nag/scripts/assignee_no_login.py]: I moved it at the beginning and made it a constant.

**Review by calixteman — APPROVED** [review]
It makes sense so r=me.

**Pull request merged**